Sources/Widgets/Widgets3D/AngleWidget/index.js

```diff
@@ -111,7 +109,11 @@ const DEFAULT_VALUES = {
 export function extend(publicAPI, model, initialValues = {}) {
   Object.assign(model, DEFAULT_VALUES, initialValues);
 
-  vtkAbstractWidgetFactory.extend(publicAPI, model, initialValues);
+  vtkAbstractWidgetFactory.extend(publicAPI, model, {
+    ...initialValues,
+    behavior: widgetBehavior,
```

Comment:
how about letting initialValues potentially overwrite behavior and widgetState ?
Move behavior and widgetSate in a `defaultValues()` function ?